envs/Harvard-Cannon/config.harvard-cannon.global_inv.yml

```diff
@@ -182,7 +181,7 @@ AIRS: false
 ##------------------------------------------------------------------
 
 ## Path for IMI runs and output
-OutputPath: "/n/holyscratch01/jacob_lab/$USER"
+OutputPath: "/n/holylfs05/LABS/jacob_lab/$USER"
```

Comment:
I think we should keep this set to scratch by default to prevent storage bloat of holylfs from folks testing the imi on cannon. Same for the non-global version of this file

Reply:
Good point. Now reverted in https://github.com/geoschem/integrated_methane_inversion/pull/245/commits/3322f5a751611a4e73363a6bd6b9a530bf28d1c3.